### PR TITLE
Update money info variables

### PR DESCRIPTION
**Pull request by ryanzhoudev**
Fixed money caching to work with emerald pouches

### CODE REVIEW

**Comment by Incompleteusern** [comment]
#372 has been merged, pretty sure that the todo can be implemented

**Comment by ryanzhoudev** [comment]
> #372 has been merged, pretty sure that the todo can be implemented

Oh yeah, completely forgot, thanks

File: src/main/java/com/wynntils/core/utils/ItemUtils.java [hunk 1]
```diff
@@ -131,7 +132,8 @@ public static ItemType getItemType(ItemStack item) {
                         damageValue = Integer.parseInt(values[1]);
                     }
 
-                    if (Item.getIdFromItem(item.getItem()) == i && item.getItemDamage() == damageValue) return e.getKey();
+                    if (Item.getIdFromItem(item.getItem()) == i && item.getItemDamage() == damageValue)
```
Comment (magicus) [inline src/main/java/com/wynntils/core/utils/ItemUtils.java]: I really really don't like multi-line if statements without braces. It's bad coding style. Keep the return on the same line, or better, add braces.

**Review by magicus — CHANGES_REQUESTED** [review]

File: src/main/java/com/wynntils/core/utils/ItemUtils.java (hunk 1 above)
Comment (ryanzhoudev) [reply src/main/java/com/wynntils/core/utils/ItemUtils.java]: This was done by IDE, I'll just throw in a couple braces

**Review by magicus — APPROVED** [review]

**Pull request merged**